articles/protocols/saml/saml-configuration/troubleshoot/common-saml-errors.md

```diff
@@ -50,6 +50,15 @@ This error appears if you haven't provided the necessary information to support
 3. Switch to the *IdP-Initiated* tab.
 4. Select the **Default Application** and the **Response Protocol** used by that Application, and (optionally) specify any additional parameters you want passed to the Application.
 
+::: note
+If you see this error when using a SP-initiated flow, one of the following is missing or empty:
+
+* The `RelayState` parameter
+* The `InResponseTo` attribute in the SAML response
+
+If these are missing or empty, Auth0 treats the login as idP-initiated. You can fix this error by checking your configuration to ensure that both fields are populated and returned appropriately.
```

Comment:
`IdP` vs `idP` 

Also what's the threshold for panel vs note? Just wondering if this is over it or not, but I haven't checked contributing doc to see if we've defined a limit.